src/app/therapist/manage-clients/manage-clients.component.ts

```diff
@@ -20,10 +22,12 @@ export class ManageClientsComponent implements OnInit {
    * Delete selectedClient!
    */
   deleteClient() {
-    this.clientService.deleteClient(this.selectedClient)
-      .then(() => {
-        this.selectedClient = null;
-      });
+    // this.clientService.deleteClient(this.selectedClient)
```

Comment:
If we're not using this anymore, then we can perhaps not just delete this out-commented code, but also remove the deleteClient from the clientService?
Would require testing obviously!